src/plugins/workspace/public/plugin.test.ts

```diff
@@ -30,4 +30,22 @@ describe('Workspace plugin', () => {
     coreStart.workspaces.currentWorkspaceId$.next('foo');
     expect(coreStart.savedObjects.client.setCurrentWorkspace).toHaveBeenCalledWith('foo');
   });
+
+  it('#setup when workspace id is in url and enterWorkspace return error', async () => {
```

Comment:
```suggestion
  it('should correctly set up WorkspacePlugin when workspace ID is present in the URL', async () => {
```

The current test title does not match whats being tested for. You dont seem to be testing for an error. This is just a suggestion based on what the test seems to be doing

Reply:
Thanks, will address it in #6154 .